src/Nav.php

```diff
@@ -252,7 +252,7 @@ private function renderItem(array $item): string
         if (!isset($item['label'])) {
             throw new InvalidArgumentException('The "label" option is required.');
         }
-
+        $this->dropdown = false;
```

Comment:
Set [by default](https://github.com/yiisoft/yii-bulma/blob/master/src/Nav.php#L21)

Reply:
This variable neads reset for each item, because it sets submenu in only this item.

Reply:
Next item after dropdown, was wraped in div. ( [code](https://github.com/yiisoft/yii-bulma/blob/master/src/Nav.php#L305) )